internal/driver/glfw/window_test.go

```diff
@@ -2215,3 +2215,51 @@ func (s *safeWindow) keyPressed(w *glfw.Window, key glfw.Key, scancode int, acti
 		s.window.keyPressed(w, key, scancode, action, mods)
 	})
 }
+
```

Comment:
These two new tests don't appear to actually test anything.
The comments even talk about how the desired changes to test cannot be tested without hardware.
If that is true then I think there is no benefit to adding this

Reply:
I'll get rid of them.  It would probably be a pretty intrusive change to introduce the abstraction over glfw needed for testing, and not worth it.